src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -27,123 +27,132 @@
+                                             AbsSendTimeEngine absSendTimeEngine)
     {
         super(RTPPacketPredicate.INSTANCE);
         this.absoluteSendTimeEngine = absSendTimeEngine;
         this.observer_ = observer;
         this.interArrival = new InterArrival(90 * kTimestampGroupLengthMs,
-                kTimestampToMs,true);
+            kTimestampToMs, true);
         this.estimator = new OveruseEstimator(new OverUseDetectorOptions());
         this.detector = new OveruseDetector(new OverUseDetectorOptions());
-        this.incomingBitrate = new RateStatistics(kBitrateWindowMs,8000);
+        this.incomingBitrate = new RateStatistics(kBitrateWindowMs, 8000F);
         this.incomingBitrateInitialized = false;
         this.totalProbesReceived = 0;
         this.firstPacketTimeMs = -1;
         this.lastUpdateMs = -1;
-        logger.info("; RemoteBitrateEstimatorAbsSendTime: Instantiating.");
+        if (logger.isDebugEnabled())
+        {
+            logger.debug("RemoteBitrateEstimatorAbsSendTime: Active.");
+        }
     }
 
-    private <K,V> List<K> Keys(TreeMap<K,V> _map)
+    private <K, V> List<K> Keys(TreeMap<K, V> _map)
     {
         ArrayList<K> keys = new ArrayList<K>();
-        for(Map.Entry<K,V> entry : _map.entrySet())
+        for (Map.Entry<K, V> entry : _map.entrySet())
         {
             keys.add(entry.getKey());
         }
         return keys;
     }
 
-    private long ConvertMsTo24Bits(long timeMs)
+    public long ConvertMsTo24Bits(long timeMs)
     {
-        long time24Bits = (long)(((timeMs << kAbsSendTimeFraction) + 500) /
-                1000) & 0x00FFFFFF;
+        long time24Bits = (long) (((timeMs << kAbsSendTimeFraction) + 500) /
+            1000) & 0x00FFFFFF;
         return time24Bits;
     }
 
 
     private boolean IsWithinClusterBounds(long sendDeltaMs, Cluster clusterAggregate)
     {
-        if(clusterAggregate.count == 0)
+        if (clusterAggregate.count == 0)
         {
             return true;
         }
         double clusterMean = clusterAggregate.sendMeanMs /
-                (double)clusterAggregate.count;
-        return  Math.abs((double)sendDeltaMs - clusterMean) < 2.5f;
+            (double) clusterAggregate.count;
+        return Math.abs((double) sendDeltaMs - clusterMean) < 2.5f;
     }
 
     private void addCluster(List<Cluster> clusters, Cluster cluster)
     {
-        cluster.sendMeanMs /= (double)cluster.count;
-        cluster.recvMeanMs /= (double)cluster.count;
+        cluster.sendMeanMs /= (double) cluster.count;
+        cluster.recvMeanMs /= (double) cluster.count;
         cluster.meanSize /= cluster.count;
         clusters.add(cluster);
     }
 
     private void computeClusters(List<Cluster> clusters)
     {
         Cluster current = new Cluster();
-        long prevSendTime =  -1;
+        long prevSendTime = -1;
         long prevRecvTime = -1;
-        for(Probe probe : probes_)
+        for (Probe probe : probes_)
```

Comment:
i just realized perhaps all this is to be more consistent with the original cpp code, if so then can ignore all these